examples/cs.py

```diff
@@ -23,6 +24,12 @@ def integrant(theta):
 
     return compute_tcs_icdf(integrant, 0*units('rad'), np.pi*units('rad'), P)
 
+def compute_elastic_tcs(dcs, P):
+    def integrant(theta):
```

Comment:
I agree with you that we should be consistent in spelling "integrand" incorrectly.